awscognito/requirements.txt

```diff
@@ -2,4 +2,6 @@
 boto3
 
 # Required for handling both xlsx and csv file
+dataclasses
```

Comment:
are you using Python 3.7? You should not need to include this in requirements.txt for Python 3.7

Reply:
Oh...  Sorry, I am using Python 3.6.9

Reply:
Can you remove this line from requirements.txt since we require Python 3.7 as our dev environment? This may introduce confusion.